pkg/operator/starter.go

```diff
@@ -210,8 +213,26 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 					"v3.11.0/openshift-apiserver/svc.yaml",
 					"v3.11.0/openshift-apiserver/sa.yaml",
 					"v3.11.0/openshift-apiserver/trusted_ca_cm.yaml",
+				},
+			},
+			{
+				Files: []string{
 					"v3.11.0/openshift-apiserver/pdb.yaml",
 				},
+				ShouldCreateFn: func() bool {
+					isSno, err := guard.IsSNOCheckFnc(infraLister)()
```

Comment:
okay, so it will conditionally create the pdb.yaml based on the current plantform